README.md

```diff
@@ -105,12 +105,38 @@ For the best getting-started experience, make sure that you have a Salesforce DX
    - Which are my active scratch orgs?
    - Show me all the accounts in the org with alias my-org.
    - Deploy everything in my project to the org with alias my-org.
+   - Do you see any performance or security issues with the Apex code in my DX project?
+   - I see that my Apex code violates the pmd:ApexCRUDViolation rule; can you give me more information about this rule?
 
 1. To stop, restart, or view the MCP server configuration, run the **MCP: List Servers** command, click `Salesforce DX`, then click the appropriate option.
 
-## Configure Orgs and Toolsets
+## Configure the DX MCP Server
 
-You configure the Salesforce DX MCP Server by specifying at least one authorized org and an optional list of MCP toolsets.
+Configure the Salesforce DX MCP Server by passing arguments to the `args`option. Surround the argument name and its value each in double quotes, and separate all arguments and values with commas. Some arguments are Boolean and don't take a value. 
```

Comment:
```suggestion
Configure the Salesforce DX MCP Server by setting flags in the `args` array. Surround the flag name and its value each in double quotes, and separate all flags and values with commas. Some flags are Boolean and don't take a value. 
```

I think I agree with Josh. "flags" might be more familiar to users. Especially since they have the standard flag prefix (`--`)

Reply:
I changed it everywhere, I too agree it reads better.